presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergHiveMetadata.java

```diff
@@ -371,6 +401,147 @@ public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTab
         return TableStatisticsMaker.getTableStatistics(typeManager, constraint, handle, icebergTable);
     }
 
+    public void createView(ConnectorSession session, ConnectorTableMetadata viewMetadata, String viewData, boolean replace)
+    {
+        Map<String, String> properties = ImmutableMap.<String, String>builder()
+                .put(TABLE_COMMENT, "Presto View")
+                .put(PRESTO_VIEW_FLAG, "true")
+                .put(PRESTO_VERSION_NAME, prestoVersion)
+                .put(PRESTO_QUERY_ID_NAME, session.getQueryId())
+                .put(TABLE_TYPE_PROP, ICEBERG_TABLE_TYPE_VALUE)
+                .put(METADATA_LOCATION_PROP, "")
+                .build();
+
+        List<Column> columns = new ArrayList<>();
+        MetastoreContext metastoreContext = getMetastoreContext(session);
+        ColumnConverter columnConverter = metastoreContext.getColumnConverter();
+
+        for (ColumnMetadata column : viewMetadata.getColumns()) {
```

Comment:
@pratyakshsharma Are we adding actual column names here, to keep metadata updated in HMS for views? Or for any other reason? 
Do we know in the original commit since its adding dummy column in Table object, how does the metadata looks like there?

Reply:
So I have followed the way views are created in hive connector in presto. Ultimately, both (view from trino and presto) have the same metadata in HMS as checked in iceberg.information_schema.columns table